### PR TITLE
Cgit fix

### PR DESCRIPTION
**Pull request by michael-burke4**
See syzygy cgit pull request

### CODE REVIEW

File: cgitrc [hunk 1]
```diff
@@ -36,4 +36,4 @@ mimetype.svg=image/svg+xml
 
 virtual-root=/cgit
 
-scan-path=/var/git
+include=/etc/cgitrc.repos
```
Comment (charliemirabile) [inline cgitrc:39]: should probably add an example cgitrc.repos to this repo (maybe even one that just includes the scan-path line that was removed from the example cgitrc) and modify the example compose files in this repo to mount it into /etc/cgitrc.repos. I think it is important to keep these containers able to operate separably.

File: radius.py [hunk 2]
```diff
@@ -486,6 +486,8 @@ def handle_register(rocket):
 def handle_cgit(rocket):
     cgit_env = os.environ.copy()
     cgit_env['PATH_INFO'] = rocket.path_info.removeprefix('/cgit')
+    if (re.search("^/[\w\-\.]+/plain(/|$)", cgit_env['PATH_INFO'])):
```
Comment (charliemirabile) [inline radius.py:489]: I don't love this regex. Hard to read (imo) and I can't help but worry about edge cases (what if you had a repo named `plain` or something stupid). I think maybe just hiding the button is enough. Until we actually fix the plain feature, if you go to a url that isn't linked anywhere and it crashes I think that that is acceptable (especially given that we handle those errors more elegantly in the follow up commit).

File: radius.py [hunk 3]
```diff
@@ -500,8 +500,14 @@ def handle_cgit(rocket):
               file=sys.stderr)
         return rocket.respond(HTTPStatus.BAD_GATEWAY,
                               '<h1>502 BAD GATEWAY!</h1>')
-    outstring = str(so, 'UTF-8')
-    begin = outstring.index('\n\n')
+    try:
```
Comment (charliemirabile) [inline radius.py:503]: maybe prefer str.find to str.index with try catch

**Review by charliemirabile — CHANGES_REQUESTED** [review]

**Comment by theyoyojo** [comment]
Re-open this on [singularity](https://github.com/underground-software/singularity)

**Pull request closed without merging**